.github/ISSUE_TEMPLATE/major_change.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Major change announcement
+about: Propose a major change.
+title: "(My major change proposal)"
+labels: major-change
+assignees: ''
+
+---
+
+# Directions
+
+If you'd like to propose a major change to do to rustc, you've come to
+the right place! To do so, please write-up your proposal in this issue.
+
+Describe your major change in a sentence or two under the TL;DR section.
+
+If available, add links and more descriptions in the "Links and Details" section.
+Note: it is not expected that you write a fully fleshed out proposal there.
+Just add any information you think is important.
+
+List people already mentoring you on this change or reviewing this change in the "Mentors or Reviewers" section
+If there are none, you can also add people that you think would be a good fit.
+You can look at [the experts map](https://github.com/rust-lang/compiler-team/blob/master/content/experts/map.toml) for ideas.
+
+Oh, and please delete this section before you open the issue -- but
+keep the others!
```

Comment:
```suggestion
-->
```

Reply:
The "please" delete can be removed, as comments don't show up in the rendered view.